### PR TITLE
Anton/separate docker compose prod and dev

### PR DESCRIPTION
**Pull request by antonoberg**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -2,34 +2,41 @@
...
 	docker run -it --rm --name my-maven-project \
 		-v $(PWD):/usr/src/mymaven \
 		-v $(PWD)/m2:/root/.m2 \
 		-w /usr/src/mymaven \
 		maven:3 bash -c "mvn package"
 	make -C collections-api
-   
-up:
+
+up-prod:
```
Comment (Inkimar) [inline Makefile]: put after 'up-dev', alphabetical order

**Review by Inkimar — COMMENTED** [review]
we could remove the docker-compose.yml.prod and have the prod-environment in the docker-compose.yml file instead. This approach will make it a bit less complex.

File: Makefile (hunk 1 above)
Comment (antonoberg) [reply Makefile]: e86af6d1247ce00f1ef9ba766f4468d03c031561

**Comment by antonoberg** [comment]
renamed docker-compose.prod.yml to docker-compose.yml in 3f1ed8e53355fc1f5488eb6989d26a3c514bd002

**Review by Inkimar — APPROVED** [review]

**Pull request merged**